routes/votings/list-of-votings-api.js

```diff
@@ -0,0 +1,53 @@
+require('dotenv').config();
+const Joi = require('joi');
+const cors = require('cors');
+const express = require('express');
+const db = require("./list-of-votings-f");
+
+const REACT_JS_ROOT = process.env.REACT_JS_ROOT_URL
+const router = express.Router();
+router.use(cors({
+    origin: `${REACT_JS_ROOT}`
+  }));
+router.use(express.json());
```

Comment:
Same as above, I'm pretty sure you can apply it for the whole app.

Reply:
in future

Reply:
It's one of the easiest things to fix in this PR. I would fix it now.